share/completions/ansible.fish

```diff
@@ -0,0 +1,106 @@
+complete -c ansible -s i -l inventory -r -d "specify inventory host path (default=/etc/ansible/hosts) or comma separated host list."
+complete -c ansible -s l -l limit -r -d "further limit selected hosts to an additional pattern"
+complete -c ansible -l limit-hosts -r -d "outputs a list of matching hosts; does not execute anything else"
+complete -c ansible -s m -l module-name -r -d "module name to execute (default=command)"
+complete -c ansible -s M -l module-path -r -d "specify path(s) to module library (default=None)"
+complete -c ansible -l new-vault-password-file -f -d "new vault password file for rekey"
+complete -c ansible -s o -l one-line -d "condense output"
+complete -c ansible -l output -f -d "output file name for encrypt or decrypt; use - for stdout"
+complete -c ansible -s P --l poll -r -d "set the poll interval if using -B (default=15)"
+complete -c ansible -l syntax-check -f -d "perform a syntax check on the playbook, but do not execute it"
+complete -c ansible -s t -l tree -f -a "(find . -maxdepth 1 -type d)" -d "log output to this directory"
+complete -c ansible -l vault-password-file -d "vault password file"
+complete -c ansible -s v -l verbose -f -d "verbose mode (-vvv for more, -vvvv to enable connection debugging)"
+complete -c ansible -l version -f -d "show program's version number and exit"
+complete -c ansible -s k -l ask-pass -f -d "ask for connection password"
+complete -c ansible -l private-key -r -d "use this file to authenticate the connection"
+complete -c ansible -l key-file -r -d "use this file to authenticate the connection"
+complete -c ansible -s u -l user -f -d "connect as this user (default=None)"
+complete -c ansible -s c -l connection -f -d "connection type to use (default=smart)"
+complete -c ansible -s T -l timeout -f -d "override the connection timeout in seconds (default=10)"
+complete -c ansible -l ssh-common-args -f -d "specify common arguments to pass to sftp/scp/ssh (e.g. ProxyCommand)"
+complete -c ansible -l sftp-extra-args -f -d "specify extra arguments to pass to sftp only (e.g. -f, -l)"
+complete -c ansible -l scp-extra-args -f -d "specify extra arguments to pass to scp only (e.g. -l)"
+complete -c ansible -l ssh-extra-args -f -d "specify extra arguments to pass to ssh only (e.g. -R)"
+complete -c ansible -s s -l sudo -f -d "run operations with sudo (nopasswd) (deprecated, use become)"
+complete -c ansible -s U -l sudo-user -r -f -d "desired sudo user (default=root) (deprecated, use become)"
+complete -c ansible -s S -l su -f -d "run operations with su (deprecated, use become)"
+complete -c ansible -s R -l su-user -r -f -d "run operations with su as this user (default=root) (deprecated, use become)"
+complete -c ansible -s b -l become -f -d "run operations with become (does not imply password prompting)"
+complete -c ansible -l become-method -r -f -d "privilege escalation method to use (default=sudo), valid choices: [ sudo | su | pbrun | pfexec | runas | doas | dzdo ]"
```

Comment:
Give the valid choices via `-a "sudo su pbrun..."` instead of in the description.